client/src/ingredients/examples.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+function RenderSources({ sources }) {
+  return (
+    <>
+      {sources.html && <h4>HTML</h4>}
+      {/* <code> inside a <pre>?? Is that necessary? */}
+      {sources.html && (
+        <pre>
+          <code>{sources.html}</code>
+        </pre>
+      )}
+
+      {sources.css && <h4>CSS</h4>}
+      {sources.css && (
+        <pre>
+          <code>{sources.css}</code>
+        </pre>
+      )}
+
+      {sources.js && <h4>JavaScript</h4>}
+      {sources.js && (
+        <pre>
+          <code>{sources.js}</code>
+        </pre>
+      )}
+    </>
+  );
+}
+
+function RenderLiveSample({ example }) {
+  const srcdoc = `<html>
+  <head>
+      <meta charset="utf-8">
+      <style type="text/css">${example.sources.css}</style>
+      <title>${example.description.title}</title>
+  </head>
+  <body>${example.sources.html}
+      <script>${example.sources.js || ""}</script>
+  </body>
+</html>`;
+  return (
+    <>
+      <h4>Result</h4>
+      <iframe
+        className="live-sample-frame sample-code-frame"
```

Comment:
Should now be a good time to remove these and start afresh?

Reply:
I think they will probably need at least one class, but I don't know why they currently use 2.